queries.sql

```diff
@@ -8,3 +8,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = 'True';
 SELECT *FROM animals WHERE name != 'Gabumon';
 SELECT *FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ] You forgot to add a transaction. As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Now%2C%20take%20a%20deep%20breath%20and...%20Inside%20a%20transaction%20delete%20all%20records%20in%20the%20animals%20table%2C%20then%20roll%20back%20the%20transaction.), Please add a transaction that delete all records in the `animals` table, then roll back the transaction.